docs/features/override_container_command.md

```diff
@@ -13,3 +15,68 @@ req := ContainerRequest{
 }
 ```
 
+## Executing a command
+
+You can execute a command inside a running container, similar to a `docker exec` call:
+
+```go
```

Comment:
Instead of pasting here code snippets, we usually embed code blocks with the `<!--codeinclude-->` block.

Please take a look at how it's implemented, as it's unfortunately not documented (yet).

It basically needs:

1. in the markdown:
```md
<!--codeinclude-->
[Tab text](../path/to/file.extension) inside_block:name_of_the_block
<!--/codeinclude-->
```
2. in the file.extension
```go
// name_of_the_block {
... code snippet to be rendered in the markdown
// }
```

I'm going to use this comment as seed for an issue to include that in our docs.

Reply:
Thanks @mdelapenya. That makes sense. The thing is that the examples included in this PR are made up, don't correspond with any test available. I've seen that the those `codeinclude` chunks reference code in test suite. Want me to include the examples to the test suite? Or where should I put those examples?

Reply:
I'm gonna be proactive and I'll add that example to the test suite. I'll let you know when that's ready.

Reply:
Thanks @danvergara, much appreciated! 🙏